### ISSUE TITLE
Animation panel dropdown is not keyboard accessible

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5524]
## Bug Description

The animation type dropdown is not keyboard-accessible at all. There’s no focus-styling on either the unopened dropdown nor the active option. We need all the same keyboard-functionality as for the “regular” advanced dropdown (space, enter or arrow-down to open dropdown, auto-focus first item when opening, arrow keys to navigate in dropdown, enter/space to select option, esc to close dropdown, and auto-focus dropdown button itself on close).

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
@csossi - there's details in the PR on testing/changes: https://github.com/google/web-stories-wp/issues/5570

**Comment by csossi** [comment]
After selecting an animation type by keyboard, the cursor (ENTER on selection), my cursor moved to the items below the canvas (Page 1, delete icon, dupe icon, etc.). Shouldn't user stay within the Animation section of the design panel so that they can set duration, delay, etc.)?

https://share.getcloudapp.com/P8umemE7

**Comment by BrittanyIRL** [comment]
Thanks for the catch, @csossi - it should now return the focus to the effect dropdown when a keyboard user makes a new effect selection. This mimics the behavior of the font chooser.

**Comment by BrittanyIRL** [comment]
@csossi - sorry for the back and forth here with assignment! I initially fixed the focus to return to the select menu when you hit 'enter' as you found didn't work and the reason for sending this back, and then i reviewed Max's PR that builds off of this to keep the dropdown open when new animations are selected to make updating easier on the user. So the only true way a keyboard user will exit this menu is by esc or tab keys, both of which refocus to the animation dropdown closed.

**Comment by csossi** [comment]
Verified in QA (checking state in 5586)

**Comment by tmagalhaes-huge** [comment]
Works here.
